README.md

```diff
@@ -122,11 +135,112 @@ To install pgai from source on a PostgreSQL server:
 
    The `CASCADE` automatically installs `pgvector` and `plpython3u` extensions.
 
-### Use pgai
+### Usage of pgai
```

Comment:
Imo features should come before Installation. Installation is a bit of a detail and only relevant once you've been convinced by the feature set.